internal/pkg/cli/storage_init.go

```diff
@@ -197,6 +205,93 @@ func (o *initStorageOpts) validateServiceName() error {
 	return fmt.Errorf("service %s not found in the workspace", o.StorageSvc)
 }
 
+func (o *initStorageOpts) Execute() error {
+	params := `
```

Comment:
What do you think of moving those template to files in our `templates` folder and modify the `template` pkg so that in `storage_init` we can just init a serializer struct and call `Template()` to get the string? (Like what we did in `svc_package`)

Reply:
I think we are just trying to have a test binary out by Monday, so this is a bandaid fix for now. We'll remove this code afterwards.

Reply:
+1 what Efe said. This is the fastest way to get an S3 bucket's CF output by the tool at the moment, and I will be adding something similar for DDB in the next day or so.